sql/core/src/test/resources/sql-tests/results/ansi/string-functions.sql.out

```diff
@@ -289,25 +290,28 @@ trim
 -- !query
 SELECT btrim(encode(" xyz ", 'utf-8'))
 -- !query schema
-struct<btrim(encode( xyz , utf-8)):string>
+struct<>
 -- !query output
-xyz
+org.apache.spark.sql.AnalysisException
```

Comment:
See the discussion in #31390 (comment).
The function btrim was added for string type as per @beliefer

Reply:
In ANSI mode, String cannot cast as Binary ?

Reply:
Not for the implicit cast. 
![Screen Shot 2021-02-22 at 3 33 07 PM](https://user-images.githubusercontent.com/1097932/108676587-51714f80-749d-11eb-8f94-8f0b6f8a1ecc.png)